WordPress/src/main/java/org/wordpress/android/ui/JetpackConnectionDeeplinkActivity.java

```diff
@@ -65,22 +64,20 @@ protected void onCreate(Bundle savedInstanceState) {
         String action = getIntent().getAction();
         Uri uri = getIntent().getData();
 
-        AnalyticsUtils.trackWithDeepLinkData(AnalyticsTracker.Stat.DEEP_LINKED, action, uri);
+        AnalyticsUtils.trackWithDeepLinkData(Stat.DEEP_LINKED, action, uri);
 
         // check if this intent is started via custom scheme link
         if (Intent.ACTION_VIEW.equals(action) && uri != null) {
+            // Non-empty reason does not mean we're not connected to Jetpack
+            // - one of the errors is "already-connected"
```

Comment:
In this case, can we check the `reason`, if the "already-connected" is the only successful case, we can just say the connection was unsuccessful and maybe track the reason in the stats. That can help us debug certain issues.

Reply:
I added more helpful error messages + some tracking with reason. Should I mention somewhere the field I added?